app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
 
+    assert sum(split_integer(32, 6)) == 32
 
-def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
 
+def test_should_split_into_equal_parts_when_value_is_divisible_by_parts() -> None:
 
-def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert split_integer(6, 2) == [3, 3]
+
+
+def test_should_return_part_equals_to_a_value_when_slitting_into_one_part() -> None:
+
+    assert split_integer(8, 1)[0] == 8
```

Comment:
Don't complicate your code, this variant is better:
`split_integer(8, 1) == [8]`